src/pretix/control/templates/pretixcontrol/user/2fa_main.html

```diff
@@ -11,6 +11,24 @@ <h1>{% trans "Two-factor authentication" %}</h1>
             smartphone or a hardware token generator and that changes on a regular basis.
         {% endblocktrans %}
     </p>
+    {% if settings.PRETIX_OBLIGATORY_2FA %}
```

Comment:
I changed it to a yellow warning but left it to be displayed at all times.

Especially when removing the button to deactivate 2FA, this might be a good hint as to why the "disable"-button is missing. However, since 2FA gets deactivated automatically when the last device is deleted, this is more of a cosmetic change.